### PR TITLE
better help

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by darken99** [comment]
We have to fix flag parameters instead

**Comment by iakov-aws** [comment]
agree but this will be next release not now. For this release we can just add doc on existing

File: cid/cli.py [hunk 1]
```diff
@@ -81,15 +81,16 @@ def deploy(ctx, **kwargs):
...
      --quicksight-delete-failed-datasource (yes|no) Delete datasoruce if creation failed
      --quicksight-user TEXT                QuickSight user
      --dataset-{dataset_name}-id TEXT      QuickSight dataset id for a specific dataset
      --view-{view_name}-{parameter} TEXT   a custom parameter for a view creation, can use variable: {account_id}
      --account-map-source TEXT             csv, dummy, organization (if autodiscovery impossible)
      --account-map-file TEXT               csv file path relative to current directory (if autodiscovery impossible and csv selected as a source )
+     --update yes                          Update all existing dependancies to the latest version (Datasets and Views). Default=no
      --resources TEXT                      CID resources file (yaml)
-     --share-with-account ['yes/no']       Share dashboard with all users in the current account
+     --share-with-account (yes|no)         Share dashboard with all users in the current account
```
Comment (darken99) [inline cid/cli.py:93]: This should be flag parameter

File: cid/cli.py [hunk 2]
```diff
@@ -81,15 +81,16 @@ def deploy(ctx, **kwargs):
...
      --cur-table-name TEXT                 CUR table name
-     --quicksight-datasource-id TEXT      QuickSight Datasource ARN (if not found one with provided Athena workgroup)
+     --quicksight-datasource-id TEXT       QuickSight Datasource ARN (if not found one with provided Athena workgroup)
      --quicksight-delete-failed-datasource (yes|no) Delete datasoruce if creation failed
      --quicksight-user TEXT                QuickSight user
      --dataset-{dataset_name}-id TEXT      QuickSight dataset id for a specific dataset
      --view-{view_name}-{parameter} TEXT   a custom parameter for a view creation, can use variable: {account_id}
      --account-map-source TEXT             csv, dummy, organization (if autodiscovery impossible)
      --account-map-file TEXT               csv file path relative to current directory (if autodiscovery impossible and csv selected as a source )
+     --update yes                          Update all existing dependancies to the latest version (Datasets and Views). Default=no
```
Comment (darken99) [inline cid/cli.py:91]: This should be flag parameter

**Review by darken99 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**